components/tyk-dashboard/templates/deployment-dashboard.yaml

```diff
@@ -0,0 +1,255 @@
+{{- if .Values.enabled -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: dashboard-{{ include "tyk-dashboard.fullname" . }}
+  labels:
+    app: dashboard-{{ include "tyk-dashboard.fullname" . }}
+    chart: {{ include "tyk-dashboard.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  minReadySeconds: 5
+  strategy:
+    # indicate which strategy we want for rolling update
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 2
+      maxUnavailable: 1
+  selector:
+    matchLabels:
+      app: dashboard-{{ include "tyk-dashboard.fullname" . }}
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: dashboard-{{ include "tyk-dashboard.fullname" . }}
+        release: {{ .Release.Name }}
+      annotations:
+      # Enable backward compatibility to chart 0.8.1.
```

Comment:
do we still need this part?? since it is a new chart, I guess we do not need to provide any backward compatibility.

Reply:
the comment was a leftover, I will delete it.